<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaceProbDialogController</class>
 <widget class="QDialog" name="PlaceProbDialogController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STD Configurations</string>
  </property>
  <property name="windowIcon">
   <iconset resource="GUIResources.qrc">
    <normaloff>:/resources/resources/introduction1.png</normaloff>:/resources/resources/introduction1.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="labelModelFile">
      <property name="text">
       <string>Model File</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="labelPathToFile">
      <property name="text">
       <string>path_to_and_name_of_file</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelPlaceName">
      <property name="text">
       <string>Place name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditPlaceName">
      <property name="text">
       <string>reservoir</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelMaxTime">
      <property name="text">
       <string>Maximum Time</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditMaxTime">
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelTimeStep">
      <property name="text">
       <string>Time step size</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditTimeStep">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QRadioButton" name="radioConstRange">
      <property name="text">
       <string>Constant Range</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="label3dPlot">
      <property name="text">
       <string>output: 3-D plot + file</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="labelConstStep">
      <property name="text">
       <string>Constant step size</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEditConstStep">
      <property name="text">
       <string>0.2</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="labelConstStart">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Constant start</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="lineEditConstStart">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="labelConstEnd">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Constant end</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="lineEditConstEnd">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QRadioButton" name="radioSpecConst">
      <property name="text">
       <string>Specific Constant</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLabel" name="label2dPlot">
      <property name="text">
       <string>output: 2-D plot + file</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="labelConst">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Constant</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLineEdit" name="lineEditConst">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelDESRuns">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Simulation runs (DES)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="spinBoxDESRuns">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>500</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditPlaceName</tabstop>
  <tabstop>lineEditMaxTime</tabstop>
  <tabstop>lineEditTimeStep</tabstop>
  <tabstop>radioConstRange</tabstop>
  <tabstop>radioSpecConst</tabstop>
  <tabstop>lineEditConst</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="GUIResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PlaceProbDialogController</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PlaceProbDialogController</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditConst</sender>
   <signal>textEdited(QString)</signal>
   <receiver>radioSpecConst</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditConstStart</sender>
   <signal>textEdited(QString)</signal>
   <receiver>radioConstRange</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditConstEnd</sender>
   <signal>textEdited(QString)</signal>
   <receiver>radioConstRange</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
